roles/sro_deploy_custom_commit/tasks/deploy.yml

```diff
@@ -0,0 +1,25 @@
+- block:
+  - name: Git checkout
+    git:
+      repo: "{{ sro_git_repo }}"
+      dest: /tmp/special-resource-operator
+      version: "{{ sro_git_ref }}"
+      
+  - name: Deploy the custom version of the special resource operator
+    shell: |
```

Comment:
you can use `command` when `shell` features are not required
ansible-lint should have seen that ... maybe it doesn't pick up new files ... strange ...